BitwardenShared/Core/Vault/Models/Response/CollectionDetailsResponseModel.swift

```diff
@@ -14,6 +14,9 @@ struct CollectionDetailsResponseModel: Codable, Equatable {
     /// The collection's identifier.
     let id: String
 
+    /// Whether the collection can be managed by the user.
+    let manage: Bool
```

Comment:
❓ Do we need to worry about this being potentially missing on older self-hosted instances? If so, we could use the `@DefaultFalse` property wrapper.